### PR TITLE
created courses page

### PR DESCRIPTION
**Pull request by willyserge**
### what this PR does

- filter courses by category( available and upcoming)

### CODE REVIEW

File: src/components/course/course/index.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    }
+    &__content{
+      line-height: 1.5;
+      font-size: 0.9em;
+      padding: 15px;
+      background: #fafafa; 
+      color: #000000;
+      
+    }
+}
```
Comment (eliemugenzi) [inline src/components/course/course/index.scss:19]: Add a new line here

File: src/components/course/index.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    }
+    &__content{
+      line-height: 1.5;
+      font-size: 0.9em;
+      padding: 15px;
+      background: #fafafa; 
+      color: #000000;
+      
+    }
+}
```
Comment (eliemugenzi) [inline src/components/course/index.scss:19]: A new line is needed here

File: src/components/Search/search.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.content__search{
+    display: flex;
+    justify-content: space-evenly;
+    margin: auto 10%;
+}
```
Comment (eliemugenzi) [inline src/components/Search/search.scss:5]: New line is needed

File: src/components/Search/input.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                width: 40%;
+                height: 40%;
+                position: absolute;
+                top: 50%;
+                left: 50%;
+                transform: translate(-50%, -50%);
+            }
+        }
+    }
+}
```
Comment (eliemugenzi) [inline src/components/Search/input.scss:37]: New line here

File: src/assets/styles/pages/courses.scss [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+            }
+        }
+    }
+}
+.courses__container{
+    margin: 50px 10%;
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(225px,1fr));
+    gap : 20px;
+}
```
Comment (eliemugenzi) [inline src/assets/styles/pages/courses.scss:31]: New line here

Comment (willyserge) [reply src/assets/styles/pages/courses.scss:31]: Ok, will have to fix my ESlint i guess

Comment (eliemugenzi) [reply src/assets/styles/pages/courses.scss:31]: Just add a new line. ESLint does not work in SASS files.

Comment (willyserge) [reply src/assets/styles/pages/courses.scss:31]: @eliemugenzi  ok. didn't properly check. let me fix it

File: src/components/Search/input.scss (hunk 4 above)
Comment (eliemugenzi) [reply src/components/Search/input.scss:37]: Here as well

File: src/layouts/PostLayout.js [hunk 6]
```diff
@@ -18,6 +18,11 @@ import SEO from '../components/seo';
 library.add(fab, faCoffee);
 
 const PostLayout = ({ data }) => {
+  const styles = {
+    margin: 50,
```
Comment (eliemugenzi) [inline src/layouts/PostLayout.js:22]: You can leave this how it was. I have already worked on it.

Comment (willyserge) [reply src/layouts/PostLayout.js:22]: ok great

Comment (willyserge) [reply src/layouts/PostLayout.js:22]: ok ,great

**Comment by eliemugenzi** [comment]
It is still a draft, feel free to make it ready for reviews so that others can go through it.
> 
<sub>[View in Slack](https://africai.slack.com/archives/G012X8NTXN1/p1590673262004300)</sub>

**Comment by billyjason** [comment]
Nice work @willyserge ! can it be easy for you to add a way for identifying upcoming courses... like  not yet offered coursed so people can see they are  inactive at the moment, then we will add a way to register an interest for them? If not, don't worry we can plan it in our next meeting.

**Comment by willyserge** [comment]
@billyjason that's ok. will implement it tomorrow

**Pull request merged**